music/src/piano_genie/model.ts

```diff
@@ -60,6 +60,43 @@ function disposeState(state: LSTMState) {
   }
 }
 
+/**
+ * Root note of chord for chord-conditioned model.
+ */
+enum PitchClass {
```

Comment:
Can we reuse https://github.com/tensorflow/magenta-js/blob/master/music/src/protobuf/proto.d.ts#L390?